src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1154,6 +1202,7 @@ namespace winrt::TerminalApp::implementation
 
         const auto dispatchToElevatedWindow = ctrlPressed && !IsRunningElevated();
 
+        std::string sessionType;
```

Comment:
Given that `TraceLoggingValue` takes a `const char*` and that our values are all literal strings, why not use `const char*` here as well?